### ISSUE TITLE
Customize item view, facets, and search filters for patents collection

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#374]
Please customize item view, facets, and search filters for the patents collection, http://vtechworks.lib.vt.edu/handle/10919/72294, following the specifications in https://git.it.vt.edu/digital-research-services/VTechWorks_Documentation/wikis/Virginia_Tech_Patents. 

You might reference PR #76, which added a custom field to the item view of another collection and PR #50, which customized search filters and facets for several collections.

**Comment by alawvt** [comment]
will revisit next sprint

**Comment by alawvt** [comment]
Instructions for customizing the DSpace submission form

[DSpace Wiki - Submission User Interface](https://wiki.duraspace.org/display/DSDOC5x/Submission+User+Interface)

[DSpace Wiki How-to Guide - Alter submission input forms](https://wiki.duraspace.org/display/DSPACE/Alter+submission+input+forms)

**Comment by AnushaPillai** [comment]
Resolved a subset of this issue in Pull Request #419 This pull request resolves the customization of the facets and search filter for the patent collection

**Comment by alawvt** [comment]
@AnushaPillai, since it might take a while to customize the display of all the additional fields in the Simple Item View, it might be good to first create a separate pull request when you get the search result view to display the inventor, rather than "unknown author."

**Comment by alawvt** [comment]
@AnushaPillai, please push all your work to a branch and leave notes for Philippe to continue the customization, as needed.

**Comment by AnushaPillai** [comment]
@pgray64 I have made a new branch 'itemViewPatents' and pushed my changes to it. I had changed the item view to display additional fields for Patents. The item-view.xsl needs to be configured to suit the Patent specific changes. This can be done by specifying a condition that works on the collection id for patents. However I was having issues in getting the collection Id in the file item-view.xsl. So work was left incomplete there.
